src/pages/index.js

```diff
@@ -0,0 +1,329 @@
+import { enableValidation, settings, resetValidation} from "../scripts/validation.js";
+import "./index.css";
+import spotsLogoSrc from "../images/logo.svg";
+import spotsAvatarSrc from "../images/avatar.jpg";
+import spotsPencilSrc from "../images/pencil.svg";
+import spotsAddSrc from "../images/add.svg";
+import Api from "../utils/api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const spotsLogo = document.getElementById("spots-logo");
+if (spotsLogo) spotsLogo.src = spotsLogoSrc;
+
+const spotsPencil = document.getElementById("spots-pencil");
+if (spotsPencil) spotsPencil.src = spotsPencilSrc;
+
+const spotsAdd = document.getElementById("spots-add");
+if (spotsAdd) spotsAdd.src = spotsAddSrc;
+
+
+const api = new Api({
+    baseUrl: "https://around-api.en.tripleten-services.com/v1",
+    headers: {
+      authorization: "ea7006fd-eee9-4b90-adca-27b6c65eefb9",
+      "Content-Type": "application/json"
+    }
+  });
+
+
+//Profile Elements
+const profileNameElement = document.querySelector(".profile__name");
+const profileDescriptionElement = document.querySelector(".profile__description");
+const spotsAvatar = document.getElementById("spots-avatar");
+if (spotsAvatar) spotsAvatar.src = spotsAvatarSrc;
+
+api.getAppInfo()
+    .then(([userData]) => {
+        profileNameElement.textContent = userData.name;
+        profileDescriptionElement.textContent = userData.about;
+        spotsAvatar.src = userData.avatar;
+    
+}).catch(console.error);
+
+const cardList = document.querySelector(".cards__list");
+
+  api.getAllCards()
+  .then((cards) => {
+      cards.forEach((item) => {
+          if (!document.querySelector(`[data-card-id="${item._id}"]`)) {
```

Comment:
```js

          if (!document.querySelector(`[data-card-id="${item._id}"]`)) {

```


It's not needed  because you should render all server cards without any checking